src/index.js

```diff
@@ -31,8 +31,8 @@ class TypingAnimation extends React.Component {
   }
 
   render() {
-    const { dotStyles, style, dotColor, dotMargin, dotRadius, dotX } = this.props;
-
+    const { dotStyles, style, show, dotColor, dotMargin, dotRadius, dotX } = this.props;
+    show ? style.visibility = "visible" : style.visibility = "hidden";
```

Comment:
This mutation seems off to me @murageden. Mutating styles in the `render` function feels like an anti-pattern. 

Can we make this more functional and instead do this:

```suggestion
    if(!show) return null;
```

Reply:
My thinking is we still want to keep the space occupied by the view such that the view doesn't appear to be pushing itself in and out.

Reply:
@murageden Could you send me a loom video of what you are seeing when you get a chance? :) 

https://www.loom.com/

Reply:
You're right Adrian, this implementation is [anti-pattern](https://daveceddia.com/why-not-modify-react-state-directly/). And I feel like this feature will become redundant. The `style` prop should be used instead, to achieve the intended functionality like so: 
```jsx
import React from "react";
import { TypingAnimation } from 'react-native-typing-animation';

class Example extends React.Component {
  render() {
    return (
      <TypingAnimation 
        dotColor="black"
        dotMargin={10}
        dotAmplitude={3}
        dotSpeed={0.08}
        dotRadius={6.5}
        dotX={12}
        dotY={6}
        style={{visibility: "hidden"}}
      />
    );
  }
}
```
This is easier and more consistent with the design. What do you think?

Reply:
I made the changes tho.

Reply:
@murageden I think it is ok to have this extra `show` prop for extra convenience. Thank you for contributing 😄

Reply:
if its work no other change requre